src/services/edusharing/services/EduSharingConnectorV7.js

```diff
@@ -221,11 +227,51 @@ class EduSharingConnector {
 		}
 	}
 
+	async getPlayerForNode(nodeUuid) {
+		const url = `${ES_ENDPOINTS.RENDERER}${nodeUuid}`;
+		const options = {
+			method: 'GET',
+			url,
+			headers: {
+				Accept: 'application/json',
+			},
+		};
+		const user = Configuration.get('ES_USER');
+
+		const response = await this.eduSharingRequest(options, user);
+		const parsed = JSON.parse(response);
+		if (parsed && parsed.detailsSnippet && typeof parsed.detailsSnippet === 'string') {
+			return this.getH5Piframe(parsed.detailsSnippet);
+		}
+		throw new Unavailable(`Unexpected response from Edu-Sharing renderer.`);
+	}
+
 	validateUuid(uuid) {
 		const uuidV5Regex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
```

Comment:
The uuid for V5, V6 and V7 is the same? Then the naming of V5 on this place is wrong.
V4 is also not only this version i think, it mean all versions before too. 

Not part of your PR i know but i want to fix it to make the code more self explained.

Reply:
The V5 doesn't mean the EduConnector versions. It is only for the UUID. Furthermore, this is not part of this PR, because it comes from main-branch.